contrib/win32/win32compat/win32-utf8.c

```diff
@@ -65,5 +65,7 @@ snmprintf(char *buf, size_t len, int *written, const char *fmt, ...)
 void
 msetlocale(void)
 {
+	// allow console output of unicode characters
+	SetConsoleOutputCP(CP_UTF8);
```

Comment:
I believe this  changes console code page without user's intention ?
Users can work around by explicitly running 
`chcp 65001`

Reply:
@manojampalam Yes, it will change the console code page.  SSH mucks with window size, colors, scroll bars, titles, etc.  Is this fundamentally different?

Reply:
@NoMoreFood agree, and that's some thing we need to work in the long term to redesign and have ssh client either work on an isolated console window (not sure if possible) or restore the console settings to original state at clean up (possible but not robust). 

In the mean time, do you feel this change is necessary given the workaround that lets users manage their code pages explicitly?

Reply:
@manojampalam  Since I don't have any analytics to make an informed decision, I can't say I have a strong opinion.  My gut says that it's probably a good change to make to support international users, but it won't hurt my feelings if you don't think it's worth the risk of mucking with console settings.

Reply:
I don't think it's a problem to do this. 

> Sets the output code page used by the console associated with **the calling process.** A console uses its output code page to translate the character values written by the various output functions into the images displayed in the console window.